tests/common.go

```diff
@@ -5269,6 +5269,44 @@ func DeleteNfsSubPath(subPath string) {
 	log.FailOnError(err, fmt.Sprintf("Failed to run [%s] command on node [%s], error : [%s]", rmCmd, workerNode, err))
 }
 
+//RemoveCloudSnapshotsFromNFSLocation deletes the cloud snapshots files from NFS backup location
+func RemoveCloudSnapshotsFromNFSLocation(cloudSnapShotPath string) {
+	// Getting NFS share details from ENV variables.
```

Comment:
Can we have this as common function for this and https://github.com/portworx/torpedo/pull/1993
We can pass path and file name to delete as parameter to this function

Reply:
Having a common function for both the TC's. Addressed

Reply:
Removing the helper method as it was deemed as duplicate and unnecessary. Making use of the existing function DeleteNfsSubPath().